src/pages/Quests.vue

```diff
@@ -1,24 +1,104 @@
 <template>
   <main>
-    <h1>Quests Page not Found</h1>
+    <section class="quests">
+      <div class="quests-container">
+        
+        <div class="quest-header-container">
+            <h1 class="quests-title">Quests</h1>
+            <div class="search-box">
+              <input type="text" id="questSearch" placeholder="Search for a quest" onkeyup="questSearch()" required>
+              <img src="@/assets/images/vectors/search.png">
+            </div>
+        </div>
+        <div class="cards">
+          <Card title="Stackdriver Logging" level="Fundamental" hours="5" credits="23" steps="5"/>
```

Comment:
For the hours, credits, and steps props, change them to the Number type since you've specified them in your component with the type Number.